admin/osx/pre_install.sh.cmake

```diff
@@ -2,5 +2,6 @@
 
 # kill the old version. see issue #2044
 killall @APPLICATION_EXECUTABLE@
```

Comment:
why do we need to keep this line ?

Reply:
The `preinstall` script opportunistically tries to kill a possibly running client. However, it cannot tell whether the previous binary is named `nextcloud` (pre 3.3.0) or `Nextcloud`. In fact, it doesn`t care if there was a client installed in the first place.

In order to cater for all cases, this PR changes the `preinstall` script to just try to kill any of `nextcloud` and `Nextcloud`.

Reply:
OK